newIDE/app/src/EffectsList/index.js

```diff
@@ -110,6 +111,13 @@ export default function EffectsList(props: Props) {
 
     forceUpdate();
     onEffectsUpdated();
+
+    // An effect is always added at the end.
```

Comment:
Could you add that you're talking about a gd effect item (I though it was a React effect)

Reply:
Or maybe move the comment where the scrollToBottom is